frontend/src/container/OnboardingContainer/Onboarding.styles.scss

```diff
@@ -99,6 +99,11 @@
 	display: flex;
 	justify-content: center;
 	margin: 48px 0;
+
+	> button {
```

Comment:
Lucid icons were not getting aligned properly with the text inside buttons. This styling is to manage that. Similar to how we have handled it in the "Invite" button in Line 190.

Reply:
@amlannandy : You can use `periscope-btn` class to handle the alignment and styling 

Ex:

```
<Button
   className="periscope-btn primary"
   ...
>
    New Source
</Button>

```

Reply:
Ah got it. Have updated it now. Updated screenshot - 

<img width="504" alt="Screenshot 2024-12-04 at 8 20 39 AM" src="https://github.com/user-attachments/assets/35b1f00c-82d0-4d07-a41e-8f82d0679cea">